### DISCUSSION TITLE
Disable to send json header

### DISCUSSION THREAD

**Discussion by avoitenko** [jchristn/WatsonTcp#155] [General]
Hello.
I do sendAsync with short message: qqq
Can I disable sending json message like {"s":"Normal","len":3} ?

Is it possible to add an option to Settings for this?

**Comment by jchristn** [comment]
It is not. Framing is integral to the library. If you are doing your own framing or otherwise need TCP without framing I have two other libraries: SuperSimpleTcp and CavemanTcp.

**Reply by avoitenko** [reply]
Thanks
